src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java

```diff
@@ -404,6 +405,28 @@ private String getServerName() {
         return tmplt;
     }
 
+    /**
+     * Return the number of active nodes provisioned using this template.
+     *
+     * @param onlyNewComputers Whether or not to limit the list to nodes running on new machines.
+     */
+    public int getActiveNodesTotal(boolean onlyNewComputers) {
+        int totalServers = 0;
+        for (Server server : getRunningNodes()) {
```

Comment:
Note that `getRunningNodes()` queries openstack for servers that are created from this template in any stage of provisioning: getting ready, utilized, tearing down or waiting for it. It would be better to focus on computers in Jenkins IMO.

Reply:
Very good point, I will fix this.